src/goto-instrument/goto_instrument_parse_options.h

```diff
@@ -86,7 +86,11 @@ Author: Daniel Kroening, kroening@kroening.com
   "(undefined-function-is-assume-false)" \
   "(remove-function-body):"\
   "(splice-call):" \
-  OPT_REMOVE_CALLS_NO_BODY
+  OPT_REMOVE_CALLS_NO_BODY \
+  "(replace-function-body):" \
```

Comment:
But these options and their documentation in to `replace_function_bodies.h` to make it easier for it to be reused in different front ends.

Reply:
As a define like some of the other options?